src/HDInsight/HDInsight/Az.HDInsight.psd1

```diff
@@ -133,7 +134,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Supported Customer-managed key disk encryption.'
+        ReleaseNotes = '* Supported list hosts and restart hosts of the cluster'
```

Comment:
release note will be handled every release, please revert this line

Reply:
Reverted

---
In reply to: [438664464](https://github.com/Azure/azure-powershell/pull/12124#discussion_r438664464) [](ancestors = 438664464)